src/views/LoginView.vue

```diff
@@ -17,11 +17,42 @@
           variant="secondary"
           v-if="pendingUserStats && pendingUserStats.email"
         >
-          You have {{ pendingUserStats.numApproved > 0 ? pendingUserStats.numApproved : '' }}
-          Codex {{ pendingUserStats.numApproved === 1 ? 'Record' : 'Records' }}
-          waiting to be claimed. Log in with an Identity Provider below
-          associated with the email <strong>{{ pendingUserStats.email }}</strong> to
-          claim {{ pendingUserStats.numApproved === 1 ? 'it' : 'them' }}!
+
+          <!--
```

Comment:
Can you move all of this logic into a method or a computed property instead of having the markup be so verbose? I believe you can even return JSX directly from a method if you want to embed <strong> within the helper.

Reply:
Hah, it looks like it rendered the `strong` tag in markdown

Reply:
Good call 👍